src/WorkItemMigrator/tests/Migration.Common.Tests/Config/MigrationCommonConfigTests.cs

```diff
@@ -0,0 +1,110 @@
+            string Query = "Query";
+            string Workspace = "Workspace";
+            string EpicLinkField = "EpicLinkField";
+            string SprintField = "SprintField";
+            int DownloadOptions = 1;
+            int BatchSize = 1;
+            string LogLevel = "LogLevel";
+            string AttachmentsFolder = "AttachmentsFolder";
+            string UserMappingFile = "UserMappingFile";
+            string BaseAreaPath = "BaseAreaPath";
+            string BaseIterationPath = "BaseIterationPath";
+            bool IgnoreFailedLinks = true;
+            string ProcessTemplate = "ProcessTemplate";
+            string[] RenderedFields = new string[0];
+            bool UsingJiraCloud = true;
+
+            FieldMap fieldMap = new FieldMap();
+            TypeMap typeMap = new TypeMap();
+            LinkMap linkMap = new LinkMap();
+            List<CharReplaceRule> CharReplaceMap = new List<CharReplaceRule>();
+
+            string jsonString = "{"
+                + string.Format("\"source-project\": \"{0}\",", SourceProject)
+                + string.Format("\"target-project\": \"{0}\",", TargetProject)
+                + string.Format("\"query\": \"{0}\",", Query)
+                + string.Format("\"workspace\": \"{0}\",", Workspace)
+                + string.Format("\"epic-link-field\": \"{0}\",", EpicLinkField)
+                + string.Format("\"sprint-field\": \"{0}\",", SprintField)
+                + string.Format("\"download-options\": \"{0}\",", DownloadOptions)
+                + string.Format("\"batch-size\": \"{0}\",", BatchSize)
+                + string.Format("\"log-level\": \"{0}\",", LogLevel)
+                + string.Format("\"attachment-folder\": \"{0}\",", AttachmentsFolder)
+                + string.Format("\"user-mapping-file\": \"{0}\",", UserMappingFile)
+                + string.Format("\"base-area-path\": \"{0}\",", BaseAreaPath)
+                + string.Format("\"base-iteration-path\": \"{0}\",", BaseIterationPath)
+                + string.Format("\"ignore-failed-links\": \"{0}\",", IgnoreFailedLinks)
+                + string.Format("\"field-map\": {0},", JsonConvert.SerializeObject(fieldMap))
+                + string.Format("\"process-template\": \"{0}\",", ProcessTemplate)
+                + string.Format("\"type-map\": {0},", JsonConvert.SerializeObject(typeMap))
+                + string.Format("\"link-map\": {0},", JsonConvert.SerializeObject(linkMap))
+                + string.Format("\"rendered-fields\": {0},", JsonConvert.SerializeObject(RenderedFields))
+                + string.Format("\"using-jira-cloud\": \"{0}\",", UsingJiraCloud)
+                + string.Format("\"sprint-char-replace-map\": {0}", JsonConvert.SerializeObject(CharReplaceMap))
+                + "}";
+
+
+            ConfigJson configJson = JsonConvert.DeserializeObject<ConfigJson>(jsonString);
+
+            Assert.That(configJson.SourceProject, Is.EqualTo(SourceProject));
```

Comment:
This is bad practise to have asserts one after another if it fails in the middle you have no clue what will happen to the next ones.

Use:

wrap those into  Assert.Multipe(()=> {
HEre go my asserts
});

Reply:
What are we testing here ? that Json conversion work ?

Reply:
Wrapped all multiple asserts into Assert.Multiple

Reply:
Its fine, just wanted to confirm, if that was the case :)